tests/integrations/openai/test_openai.py

```diff
@@ -0,0 +1,157 @@
+from openai import OpenAI, Stream
+from openai.types import CompletionUsage, CreateEmbeddingResponse, Embedding
+from openai.types.chat import ChatCompletion, ChatCompletionMessage, ChatCompletionChunk
+from openai.types.chat.chat_completion import Choice
+from openai.types.chat.chat_completion_chunk import ChoiceDelta, Choice as DeltaChoice
+from openai.types.create_embedding_response import Usage as EmbeddingTokenUsage
+
+from sentry_sdk import start_transaction
+from sentry_sdk.integrations.openai import OpenAIIntegration
+
+try:
+    from unittest import mock  # python 3.3 and above
+except ImportError:
+    import mock  # python < 3.3
```

Comment:
We can just use `from unittest import mock` because it will never run in old pythons (and we drop support for older pythons in general with the upcoming SDK 2.0)